pkg/server/version.go

```diff
@@ -17,29 +17,30 @@ limitations under the License.
 package server
 
 import (
-	"fmt"
-
 	"golang.org/x/net/context"
 	"k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime"
+
+	"github.com/kubernetes-incubator/cri-containerd/pkg/version"
 )
 
 const (
-	containerName        = "containerd"
+	// For now, containerd and runc are bundled with cri-containerd, cri-containerd
+	// version is more important to us.
+	// TODO(random-liu): Figure out how to package cri-containerd and containerd,
+	// and how to version it. We still prefer calling the container runtime "containerd",
+	// but we care both the cri-containerd version and containerd version.
+	containerName        = "cri-containerd"
 	containerdAPIVersion = "0.0.0"
 	// kubeAPIVersion is the api version of kubernetes.
 	kubeAPIVersion = "0.1.0"
 )
 
 // Version returns the runtime name, runtime version and runtime API version.
 func (c *criContainerdService) Version(ctx context.Context, r *runtime.VersionRequest) (*runtime.VersionResponse, error) {
-	resp, err := c.client.Version(ctx)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get containerd version: %v", err)
-	}
 	return &runtime.VersionResponse{
 		Version:        kubeAPIVersion,
 		RuntimeName:    containerName,
-		RuntimeVersion: resp.Version,
+		RuntimeVersion: version.CRIContainerdVersion,
 		// Containerd doesn't have an api version now.
 		RuntimeApiVersion: containerdAPIVersion,
```

Comment:
Why don't we use the CRI version here?

Reply:
? We don't have api version. Currently it's just 0.0.0